internal/planmodifiers/planmodifiers.go

```diff
@@ -0,0 +1,61 @@
+package planmodifiers
```

Comment:
This file is not required as attribute plan modifiers are ignored if they appear on attributes in data sources or providers (see [docs](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/tfsdk#Attribute)).